### ISSUE TITLE
Invalid spec, no package name found: <NULL>

### ISSUE THREAD

**Issue by RiseT** [mamba-org/mamba#2414]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Search tried in issue tracker

Yes

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not reproducible with Conda

### Describe your issue

Not sure what's the actual cause of this; just putting this in here in case it is something the devs are unaware of, as it looks like an uncaught ("unexpected") error.

Steps:
- Create conda env
- Install Python 3.11
- Try installing zipline-reloaded (current version, i. e. 2.4, which somehow doesn't want to install with Python 3.11)

### mamba info / micromamba info

```shell
active environment : zl_test
    active env location : /home/rise/mambaforge/envs/zl_test
            shell level : 2
       user config file : /home/rise/.condarc
 populated config files : /home/rise/mambaforge/.condarc
                          /home/rise/.condarc
          conda version : 23.1.0
    conda-build version : not installed
         python version : 3.11.0.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.35=0
                          __linux=5.15.90.1=0
                          __unix=0=0
       base environment : /home/rise/mambaforge  (writable)
      conda av data dir : /home/rise/mambaforge/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/rise/mambaforge/pkgs
                          /home/rise/.conda/pkgs
       envs directories : /home/rise/mambaforge/envs
                          /home/rise/.conda/envs
               platform : linux-64
             user-agent : conda/23.1.0 requests/2.28.2 CPython/3.11.0 Linux/5.15.90.1-microsoft-standard-WSL2 ubuntu/20.04.1 glibc/2.35
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### Logs

```shell
Invalid spec, no package name found: <NULL>

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/rise/mambaforge/lib/python3.11/site-packages/conda/exceptions.py", line 1124, in __call__
        return func(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^
      File "/home/rise/mambaforge/lib/python3.11/site-packages/mamba/mamba.py", line 941, in exception_converter
        raise e
      File "/home/rise/mambaforge/lib/python3.11/site-packages/mamba/mamba.py", line 934, in exception_converter
        exit_code = _wrapped_main(*args, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/home/rise/mambaforge/lib/python3.11/site-packages/mamba/mamba.py", line 892, in _wrapped_main
        result = do_call(parsed_args, p)
                 ^^^^^^^^^^^^^^^^^^^^^^^
      File "/home/rise/mambaforge/lib/python3.11/site-packages/mamba/mamba.py", line 754, in do_call
        exit_code = install(args, parser, "install")
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/home/rise/mambaforge/lib/python3.11/site-packages/mamba/mamba.py", line 560, in install
        print(solver.explain_problems())
              ^^^^^^^^^^^^^^^^^^^^^^^^^
    RuntimeError: Invalid spec, no package name found: <NULL>

`$ /home/rise/mambaforge/bin/mamba install zipline-reloaded`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=zl_test
                CONDA_EXE=/home/rise/mambaforge/bin/conda
             CONDA_PREFIX=/home/rise/mambaforge/envs/zl_test
           CONDA_PREFIX_1=/home/rise/mambaforge
    CONDA_PROMPT_MODIFIER=(zl_test)
         CONDA_PYTHON_EXE=/home/rise/mambaforge/bin/python
               CONDA_ROOT=/home/rise/mambaforge
              CONDA_SHLVL=2
           CURL_CA_BUNDLE=<not set>
               LD_PRELOAD=<not set>
                     PATH=/home/rise/mambaforge/envs/zl_test/bin:/home/rise/mambaforge/condabin:
                          /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/game
                          s:/usr/local/games:/usr/lib/wsl/lib:/mnt/c/Program Files/WindowsApps/M
                          icrosoftCorporationII.WindowsSubsystemForLinux_1.1.3.0_x64__8wekyb3d8b
                          bwe:/mnt/c/Program Files/Broadcom/Broadcom 802.11 Network
                          Adapter:/mnt/c/Program Files (x86)/NVIDIA
                          Corporation/PhysX/Common:/mnt/c/Program Files (x86)/Common Files/Oracl
                          e/Java/javapath:/mnt/c/Windows/system32:/mnt/c/Windows:/mnt/c/Windows/
                          System32/Wbem:/mnt/c/Windows/System32/WindowsPowerShell/v1.0:/mnt/c/Wi
                          ndows/System32/OpenSSH:/mnt/c/ProgramData/chocolatey/bin:/mnt/c/Progra
                          m Files/dotnet:/mnt/c/Program Files/Calibre2:/mnt/c/Program Files
                          (x86)/Smart Projects/DVDVob2Mpg:/mnt/c/Program Files (x86)/AOMEI/AOMEI
                          Backupper/7.0.0:/mnt/c/Program Files/Go/bin:/mnt/c/Program Files/Git/c
                          md:/mnt/c/Users/Rise/AppData/Local/Microsoft/WindowsApps:/mnt/c/Users/
                          Rise/AppData/Local/Programs/Microsoft VS Code/bin:/snap/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : zl_test
    active env location : /home/rise/mambaforge/envs/zl_test
            shell level : 2
       user config file : /home/rise/.condarc
 populated config files : /home/rise/mambaforge/.condarc
                          /home/rise/.condarc
          conda version : 23.1.0
    conda-build version : not installed
         python version : 3.11.0.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.35=0
                          __linux=5.15.90.1=0
                          __unix=0=0
       base environment : /home/rise/mambaforge  (writable)
      conda av data dir : /home/rise/mambaforge/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /home/rise/mambaforge/pkgs
                          /home/rise/.conda/pkgs
       envs directories : /home/rise/mambaforge/envs
                          /home/rise/.conda/envs
               platform : linux-64
             user-agent : conda/23.1.0 requests/2.28.2 CPython/3.11.0 Linux/5.15.90.1-microsoft-standard-WSL2 ubuntu/20.04.1 glibc/2.35
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
```

### ~/.condarc

```yaml
channels:
  - conda-forge
channel_priority: strict
```

**Comment by AntoinePrv** [comment]
Thanks for reporting. I'm going to assume this is fixed by #2423. Feel free to reopen if you still encounter the issue with `(micro)mamba>1.4.1` (the latest version).

> Try installing zipline-reloaded (current version, i. e. 2.4, which somehow doesn't want to install with Python 3.11)

It looks like `zipline-reloaded` is not made `noarch: python` which means it has to be built for every version of Python, which has not happened yet for Python 3.11.
From a quick look, it seems unnecessary. You could ask the maintainer to make the package `noarch: python` there:
https://github.com/conda-forge/zipline-reloaded-feedstock

**Comment by wholtz** [comment]
I downloaded a micromamba build from after #2423 and I'm still seeing the same behavior I  described in #2426, including the error:
```
critical libmamba Invalid spec, no package name found: <NULL>
```
